### DISCUSSION TITLE
can we have the Postgres collector yml file example

### DISCUSSION THREAD

**Discussion by saranggm** [burningalchemist/sql_exporter#385] [Q&A] [answered]

**Comment by burningalchemist** [comment, marked as answer]
Hi @saranggm, I guess you can have a look at the example here: https://github.com/burningalchemist/sql_exporter/blob/25c6746f46853309ec7f0d5954b6ee67602ba20d/helm/ci/postgresql-values.yaml#L12-L38

There's no defaults as `sql_exporter` is a database-agnostic exporter, so it's up to the user to define what they want to query, be it system or user tables.
